ci-operator/step-registry/cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/custom-cidr/cucushift-installer-custom-cidr-ref.yaml",
```

Comment:
I think this step can be shared by others, not only for QE, so suggest to move the step folder to `ci-operator/step-registry/ovn/conf/custom-cidr`.

Reply:
ok, good. I will move this folder.